visualization/include/pcl/visualization/interactor_style.h

```diff
@@ -250,6 +260,11 @@ namespace pcl
           modifier_ = modifier;
         }
 
+        /** \brief ID used to fetch/display the look up table on the visualizer
+         * It should be set by PCLVisualizer \ref setLookUpTableID
+         * @note If empty, a random actor added to the interactor will be used */
+        std::string lut_actor_id_;
```

Comment:
`PCLVisualizer` is a [friend](https://github.com/VictorLamoine/pcl/blob/visualizer_lut/visualization/include/pcl/visualization/interactor_style.h#L394), so will be able access this even if it is made private. I would strongly prefer to not have public member fields.